app/assets/stylesheets/application.scss

```diff
@@ -35,6 +36,10 @@
   @import 'character_show_pages';
 
 //the default background image
+  #comments {
```

Comment:
Avoid using id selectors<br>Line should be indented 0 spaces, but was indented 2 spaces